Client/Frontend/Intro/IntroViewModel.swift

```diff
@@ -50,29 +50,15 @@ struct IntroViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
     }
 
     func getInfoModel(cardType: IntroViewModel.InformationCards) -> OnboardingModelProtocol? {
-        switch (cardType, isv106Version) {
```

Comment:
nit: `isv106Version` could be renamed to `isFeatureEnabled`

Reply:
So, this is part of the protocol, which means I'd have to update it everywhere. Is that something we're good with?

Reply:
yes I think it makes sense because now the flags are controlling the feature and they are not related to v106 anymore

Reply:
will do! :)